overops-java-bullshifier/README.MD

```diff
@@ -0,0 +1,72 @@
+
+## Install Chart
+```console
+# Helm 3 (Updating the fields to install collector and server)
+$ helm install -n=overops --set overops.collectorHost=[YOUR-COLLECTOR-HOSTNAME] myoverops overops/overops-event-generator
+```
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Executing Java Bullshifier as a Job
+TBD 
+
+## Uninstall Chart
+
+```console
+# Helm 3
+$ helm uninstall [RELEASE_NAME]
+```
+
+This removes all the Kubernetes components associated with the chart and deletes the release.
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+## Configuration
+
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), or run these configuration commands:
+
+```console
+
+# Helm 3
+$ helm show values overops/overops-java-bullshifier
+```
+
+You may similarly use the above configuration commands on each chart [dependency](#dependencies) to see it's configurations.
+
+The following table lists the configurable parameters of the OverOps chart and their default values.
+
+| Parameter                                    | Description                                                                                  | Default                           |
+| -------------------------------------------- | -------------------------------------------------------------------------------------------- | ----------------------------------|
+| `overops.collectorHost`                      | Collector hostname or K8s Service Name                                                       | `nil`                             |
+| `overops.collectorPort`                      | Collector port                                                                               | `6060`                            |
+| `overops.applicationName`                    | Application Name to display on OverOps (Automatically detected if not set)                   | `nil`                             |
+| `overops.deploymentName`                     | Deployment Name to display on OverOps (Automatically detected if not set)                    | `nil`                             |
+| `overops.serverName`                         | Server Name to display on OverOps (Automatically detected if not set)                        | `nil`                             |
+| `overops.job.enabled`                        | Enables a k8s Job Workload for the Event Generator                                           | `false`                           |
+| `overops.job.color`                          | The plan of the Bulshifier - affect on how intense will be the load on the application - (Options: white/yellow/red/black) .                                      | `white`                              |
```

Comment:
Looks like the value is actually:

`overops.color` instead of `overops.job.color`

Also would add the duration settings to the readme which are part of the "overops" section

Reply:
Done